assets/css/styles.css

```diff
@@ -721,14 +742,275 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
   }
 }
 
+/*
+  ADD PROJECT POPUP CSS
+  - Add the Popup class (same as the html file and style it)
+  --  Add .popup button and .popup h1, h2, h3 and p to respective
+      class declarations in the different sections
+  - Add the class open-popup (to be added in the javascript open function)
+*/
+
+.popup-modal {
+  position: absolute;
+  top: 0%;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: var(--image-placeholder-color);
+  z-index: 100;
+}
+
+.project-popup {
+  width: 100%;
+  position: absolute;
+  background: var(--image-placeholder-color1);
+  border-radius: 1%;
+  top: 0;
+  left: 0;
+  transform: translate(-250%, -250%) scale(0.1);
+  text-align: center;
+  padding: 0;
+  transition: transform 0.4s, visibility 0.4s, top 0.4s;
+  visibility: visible;
+}
+
+.project-popup-desktop-img {
+  width: 100%;
+  height: 60%;
+  margin-top: 3%;
+  border-radius: 1%;
+  border: 1px solid white;
+}
+
+.project-popup-mobile-img {
+  width: 100% !important;
+  height: 40%;
+  margin-top: 3%;
+  border-radius: 1%;
+  border: 1px solid white;
+}
+
+/*
+.project-popup-desktop-img img,
+.project-popup-mobile-img img {
+  width: 100% !important;
+  height: 100% !important;
+}
+*/
+
+.open-project-popup {
+  width: 100%;
+  height: 97vh;
+  top: 3vh;
+  left: 0%;
+  transform: translate(-250%, -250%) scale(0.1);
+  position: fixed;
+  visibility: hidden;
+  z-index: 1000;
+  background-color: var(--image-placeholder-color);
+}
+
+.open-project-popup .active {
+  visibility: visible;
+}
+
+.project-card-js {
+  position: fixed;
+  width: 80%;
+  height: 80vh;
+  top: 10%;
+  left: 10%;
+  background-color: white;
+  z-index: 900;
+}
+
+/*
+  STYLING THE JS CONTAINER
+*/
+
+.js-button-close {
+  position: absolute;
+  width: 5%;
+  height: 5%;
+  top: 0%;
+  right: 5%;
+  margin: 0;
+  border: 1px solid var(--image-placeholder-color);
+  border-radius: 5px;
+  background: url(../../images/popup/close.svg) no-repeat center;
+  align-self: center;
+}
+
+.js-text-container {
+  width: 90%;
+  height: 100%;
+  padding: 0.1% 5%;
+  margin-top: 5%;
+}
+
+.js-title-div {
+  width: 100%;
+  height: 10%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.js-mobile-link-list {
+  width: 100%;
+  height: 10%;
+  display: flex;
+  justify-content: center;
+  flex-wrap: nowrap;
+}
+
+.js-liveLink-img {
+  width: 20%;
+  height: 20%;
+  background: url(../../images/popup/icon-export.svg) no-repeat center;
+}
+
+.js-sourceLink-img {
+  width: 100%;
+  height: 100%;
+  background: url(../../images/popup/icon-gitHub.svg) no-repeat center;
+}
+
+.js-liveLink-button {
+  /*
+  width: 20%;
+  height: 20%;
+  */
+  background: url(../../images/popup/button-see-live.svg) no-repeat center;
+  background-size: 100%;
+}
```

Comment:
Kindly ensure that the live and GitHub links show up as represented by the design. This also applies to the languages and description.

Your design:
![image](https://user-images.githubusercontent.com/20554583/233500213-bac38901-81d1-4e9c-9892-c802c36808cc.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/233500110-f39f52c1-d964-40cd-932a-7987fc7347d4.png)

Reply:
Button fixed @emmiiorji